client-templates/gitlab/accept.json.sample

```diff
@@ -163,23 +271,41 @@
           "queryParam": "file_path",
           "values": [
             "**/package.json",
+            "**%2Fpackage.json",
             "**/yarn.lock",
+            "**%2Fyarn.lock",
             "**/package-lock.json",
+            "**%2Fpackage-lock.json",
             "**/Gemfile",
+            "**%2FGemfile",
             "**/Gemfile.lock",
+            "**%2FGemfile.lock",
             "**/pom.xml",
+            "**%2Fpom.xml",
             "**/requirements.txt",
+            "**%2Frequirements.txt",
             "**/build.gradle",
+            "**%2Fbuild.gradle",
             "**/build.sbt",
+            "**%2Fbuild.sbt",
             "**/.snyk",
+            "**%2F.snyk",
             "**/packages.config",
+            "**%2Fpackages.config",
             "**/*.csproj",
+            "**%2F*.csproj",
             "**/*.vbproj",
+            "**%2F*.vbproj",
             "**/*.fsproj",
+            "**%2F*.fsproj",
             "**/project.json",
+            "**%2Fproject.json"
             "**/Gopkg.toml",
+            "**%2FGopkg.toml",
             "**/Gopkg.lock",
-            "**/vendor.json"
+            "**%2FGopkg.lock",
+            "**/vendor.json",
+            "**%2Fvendor.json"
```

Comment:
This is slightly mismatching to the other `vendor.json` rules that insist on `/vendor/vendor.json` paths. This one allows `/foo/bar/vendor.json` as well. This doesn't relate to this specific PR, but we should align on one approach in all rules.

Reply:
right, will raise that with Boost